index.js

```diff
@@ -57,7 +58,8 @@ const _splitToStream = (outStreamCreate, fileStream, partitionStreamSize, callba
 			currentOutStream.write(chunk);
 			currentFileSize += chunk.length;
 
-			if(currentFileSize == partitionStreamSize) {
+			// In objectMode one object is read from the stream, it could be that the size is (a bit) bigger than the partition size
+			if(currentFileSize >= partitionStreamSize) {
```

Comment:
I think we should make sure to continue to enforce an actual "maxFileSize". To do this, we can allow files to be slightly smaller than the max file size by adding the object to the next output stream if it could exceed the current output stream.

One edge case to consider is what happens if the maxFileSize is chosen such that 1 object cannot fit inside an output file (in that case, we should throw noting that the user input was invalid).